python_modules/dagster/dagster_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -2832,6 +2835,50 @@ def test_asset_wipe(self, storage, instance, test_run_id):
         assert len(asset_keys) == 1
         assert storage.has_asset_key(AssetKey("asset_1"))
 
+    def test_asset_wiped_event(self, instance):
+        @asset
+        def asset_to_wipe():
+            return 1
+
+        materialize([asset_to_wipe], instance=instance)
+        materializations = instance.fetch_materializations(asset_to_wipe.key, limit=100).records
+        assert len(materializations) == 1
+
+        instance.wipe_assets([asset_to_wipe.key])
+        materializations = instance.fetch_materializations(asset_to_wipe.key, limit=100).records
+        assert len(materializations) == 0
+        wipe_events = instance.get_event_records(
+            EventRecordsFilter(event_type=DagsterEventType.ASSET_WIPED)
+        )
+        assert len(wipe_events) == 1
+        assert wipe_events[0].event_log_entry.dagster_event.asset_key == AssetKey("asset_to_wipe")
+        assert wipe_events[0].event_log_entry.dagster_event_type == DagsterEventType.ASSET_WIPED
+        assert wipe_events[0].event_log_entry.dagster_event.asset_wiped_data.partition_keys is None
+
+    def test_asset_partitioned_wiped_event(self, instance):
+        @asset(partitions_def=StaticPartitionsDefinition(["a", "b", "c"]))
+        def asset_to_wipe():
+            return 1
+
+        materialize([asset_to_wipe], instance=instance, partition_key="a")
+        materializations = instance.fetch_materializations(asset_to_wipe.key, limit=100).records
+        assert len(materializations) == 1
+
+        try:
+            instance.wipe_asset_partitions(asset_to_wipe.key, ["a"])
+        except NotImplementedError:
+            if not self.can_wipe_asset_partitions():
+                pytest.skip("wiping asset partitions is not supported for this storage")
+            raise
```

Comment:
Can we check can_wipe_asset_partitions beforehand at the start of the test instead of trying and catching the NotImplementedError?

Reply:
yeah we can. I wanted to have it test that event still gets emitted though, which means i shouldn't call `skip`, just ignore the error instead